packages/expo/src/winter/fetch/convertFormData.ts

```diff
@@ -30,6 +30,20 @@ export async function convertFormDataAsync(
       results.push(entry.file.bytes());
     } else if ('blob' in entry) {
       results.push(new Uint8Array(await blobToArrayBufferAsync(entry.blob)));
+    } else if (entry._data?.blobId != null) {
+      // When `FormData.getParts()` is called, React Native will use spread operator to copy the object and loose the Blob type info.
+      // We should find the original Blob instance from the `FormData._parts` internal properties.
+      // @ts-expect-error: react-native's proprietary Blob type
+      const blob = formData._parts?.find(
```

Comment:
The object is `[partName, blob]`, so I think the variable here should not be named "blob":

```suggestion
      const formDatum = formData._parts?.find(
```